`dagu enqueue` shows go panic instead of proper error message when receiving no dag file.

Running `dagu enqueue` without actually giving it a workflow file is obviously wrong and should cause the program to quit with a proper error message. At the moment, it simply causes a panic and prints a stack trace:

```
# /opt/dagu/dagu enqueue                                                  
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/dagucloud/dagu/internal/cmd.loadDAGWithParams(0x3?, {0x876dd80?, 0x5?, 0x158559707b50?}, 0x98?)
        /home/runner/work/dagu/dagu/internal/cmd/start.go:299 +0xaf1
github.com/dagucloud/dagu/internal/cmd.runEnqueue(0x158559458840, {0x876dd80, 0x0, 0x0})
        /home/runner/work/dagu/dagu/internal/cmd/enqueue.go:61 +0x2f2
github.com/dagucloud/dagu/internal/cmd.NewCommand.func1(0x158559210c08, {0x876dd80, 0x0, 0x0})
        /home/runner/work/dagu/dagu/internal/cmd/context.go:879 +0x32e
github.com/spf13/cobra.(*Command).execute(0x158559210c08, {0x876dd80, 0x0, 0x0})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.2/command.go:1015 +0xb14
github.com/spf13/cobra.(*Command).ExecuteC(0x86f5be0)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.2/command.go:1148 +0x465
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.2/command.go:1071
main.main()
        /home/runner/work/dagu/dagu/cmd/main.go:28 +0x1a
```

--- a/internal/cmd/enqueue.go
+++ b/internal/cmd/enqueue.go
@@ -18,6 +18,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// Enqueue returns the cobra command for queueing a DAG-run.
 func Enqueue() *cobra.Command {
 	return NewCommand(
 		&cobra.Command{

--- a/internal/cmd/enqueue.go
+++ b/internal/cmd/enqueue.go
@@ -29,6 +29,7 @@ Examples:
 	dagu enqueue --run-id=run_id my_dag -- P1=foo P2=bar
 	dagu enqueue --name my_custom_name my_dag.yaml -- P1=foo P2=bar
 `,
+			Args: cobra.MinimumNArgs(1),
 		}, enqueueFlags, runEnqueue,
 	)
 }

--- a/internal/cmd/enqueue_test.go
+++ b/internal/cmd/enqueue_test.go
@@ -8,6 +8,7 @@ import (
 
 	"github.com/dagucloud/dagu/internal/cmd"
 	"github.com/dagucloud/dagu/internal/test"
+	"github.com/stretchr/testify/require"
 )
 
 func TestEnqueueCommand(t *testing.T) {
@@ -57,4 +58,16 @@ steps:
 			th.RunCommand(t, cmd.Enqueue(), tc)
 		})
 	}
+}
+
+func TestEnqueueCommand_RequiresDAGDefinition(t *testing.T) {
+	t.Parallel()
+
+	th := test.SetupCommand(t)
+
+	err := th.RunCommandWithError(t, cmd.Enqueue(), test.CmdTest{
+		Args: []string{"enqueue"},
+	})
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "requires at least 1 arg")
 }

